File: autoskola-central-back/src/main/java/hr/autoskola/model/Instruktor.java
package hr.autoskola.model;

import java.util.List;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@AllArgsConstructor
@NoArgsConstructor
@Getter
@Setter
public class Instruktor {
	
	private Integer instruktorId;
	private Integer korisnikId;
	private Integer brojSlobodnihMjesta;
	
	private Korisnik korisnik;
	private List<Vozilo> vozila;

}

File: autoskola-central-front/src/app/core/models/instruktor.ts
import { Korisnik } from "./korisnik";
import { Vozilo } from "./vozilo";

export class Instruktor{

    instruktorId : number | undefined;
    korisnikId : number | undefined;
    brojSlobodnihMjesta!: number;

    korisnik : Korisnik  = new Korisnik();
    vozila !: Vozilo[] ;
}